### ISSUE TITLE
UI does not work well on mobile screens

### ISSUE THREAD

**Issue by curranConcepts** [curranConcepts/profit-prophet#1]
#### Description
Some of the styled elements used from Daisy UI are not rendering properly on mobile sized screens. In addition to this the buttons and modals from Daisy UI seem to be missing Accessibility attributes like tabbing between inputs (cant do this at all in modals). 

#### Files
All files related to Daisy UI

#### To Reproduce
visit the site on a screen smaller than 550px wide.

#### Tasks
Include specific tasks in the order they need to be done in. Include links to specific lines of code where the task should happen at.
- [x] Remove Daisy UI and TailwindCSS
- [x] Write a custom CSS file for styling
- [x] adjust page layout if needed
